add_contributors_to_org.py

```diff
@@ -269,18 +311,16 @@ def main(token, args):
 
     parser = ArgumentParser(description=description)
 
-    parser.add_argument('repo_or_json_file',
-                        help='Name of repository in the astropy GitHub '
-                             'organization to check for contributors '
-                             'who are not yet members of the organization. '
-                             '\nAlternatively, a json file can be specified '
-                             'that includes one or more repo names and '
-                             'settings for each repo.\n\n'
-                             'NOTE: Settings in a json file override command '
-                             'line settings.')
+    parser.add_argument('organization',
+                        help="Name of the GitHub organization to check for new "
+                             "invitees.")
+
+    parser.add_argument('repos', nargs='+',
+                        help="One or more repository in this organization "
+                             "which are to be checked for new pull requests.")
 
     parser.add_argument('--num-pr', '-n', action='store',
-                        default=2, type=int,
+                        default=1, type=int,
```

Comment:
Is this a policy change?

Reply:
There has never been a policy as far as I know -- once this is ready to launch for real I'll ask coordinated package authors for input on what they want the number to be. This bot won't add anybody to repository teams or anything, it just sends an invite to the astropy organization. Once they are in the org package maintainers can decide whether to add to a team, etc.